### PR TITLE
Update agentName to Queue and Table name

### PR DESCRIPTION
**Pull request by QuanWanxx**
- Remove **agentName** config
- Add Table and Queue names for internal job and metadata.

### CODE REVIEW

File: FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs [hunk 1]
```diff
@@ -30,16 +30,28 @@ public class JobConfiguration
         public string QueueUrl { get; set; }
 
         /// <summary>
-        /// Scheduler crontab expression.
+        /// Job information table name.
         /// </summary>
-        [JsonProperty("schedulerCronExpression")]
-        public string SchedulerCronExpression { get; set; }
+        [JsonProperty("jobInfoTableName")]
+        public string JobInfoTableName { get; set; }
```
Comment (moria97) [inline FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs:36]: Shall we also update the deploy arm template to reflect this change?

Comment (QuanWanxx) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs:36]: Updated, thanks.

Comment (moria97) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs:36]: Maybe also `appsettings.json`?

Comment (QuanWanxx) [reply FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Configurations/JobConfiguration.cs:36]: Updated

**Review by moria97 — APPROVED** [review]

**Pull request merged**